mkosi/__init__.py

```diff
@@ -3293,7 +3293,10 @@ def make_tar(args: CommandLineArguments, root: str, do_run_build_script: bool, f
         if shutil.which("bsdtar") and args.distribution == Distribution.openmandriva:
             _tar_cmd = ["bsdtar", "-C", root, "-c", "-J", "--xattrs", "-f", "-", "."]
         else:
-            _tar_cmd = ["tar", "-C", root, "-c", "-J", "--xattrs", "--xattrs-include=*", "."]
+            _tar_cmd = ["tar", "-C", root, "-c", "-J", "--xattrs", "--xattrs-include=*"]
+            if args.tar_strip_selinux_context:
+                _tar_cmd.append('--xattrs-exclude=security.selinux')
+            _tar_cmd.append('.')
```

Comment:
Please run black to ensure consistent formatting (or manually fix this to use double quotes).